src/lib/stores/pagination.ts

```diff
@@ -151,21 +152,23 @@ export const outOfBounds = (
   return false;
 };
 
+export const hasId = (item: unknown): item is { id: string } => {
+  return (
+    typeof item === 'object' && Object.prototype.hasOwnProperty.call(item, 'id')
+  );
+};
+
```

Comment:
I think we already have a `has()` helper function somewhere

Reply:
👍